specification/devcenter/data-plane/Microsoft.DevCenter/preview/2023-07-01-preview/devbox.json

```diff
@@ -0,0 +1,1967 @@
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time"
+        }
+      }
+    },
+    "RemoteConnection": {
+      "description": "Provides remote connection information for a Dev Box.",
+      "type": "object",
+      "properties": {
+        "webUrl": {
+          "description": "URL to open a browser based RDP session.",
+          "type": "string"
+        },
+        "rdpConnectionUrl": {
+          "description": "Link to open a Remote Desktop session.",
+          "type": "string"
+        }
+      }
+    },
+    "PowerState": {
+      "type": "string",
+      "enum": [
+        "Unknown",
+        "Running",
+        "Deallocated",
+        "PoweredOff",
+        "Hibernated"
+      ],
+      "description": "The power states of a Dev Box.",
+      "x-ms-enum": {
+        "name": "PowerState",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Unknown",
+            "description": "The Dev Box power state is not known."
+          },
+          {
+            "value": "Running",
+            "description": "The Dev Box is running."
+          },
+          {
+            "value": "Deallocated",
+            "description": "The Dev Box is deallocated."
+          },
+          {
+            "value": "PoweredOff",
+            "description": "The Dev Box is powered off."
+          },
+          {
+            "value": "Hibernated",
+            "description": "The Dev Box is hibernated."
+          }
+        ]
+      }
+    },
+    "StopOnDisconnectConfiguration": {
+      "type": "object",
+      "description": "Stop on disconnect configuration settings for Dev Boxes created in this pool.",
+      "properties": {
+        "status": {
+          "description": "Indicates whether the feature to stop the devbox on disconnect once the grace period has lapsed is enabled.",
+          "$ref": "#/definitions/StopOnDisconnectEnableStatus"
+        },
+        "gracePeriodMinutes": {
+          "description": "The specified time in minutes to wait before stopping a Dev Box once disconnect is detected.",
+          "type": "integer",
+          "format": "int32"
+        }
+      },
+      "required": [
+        "status"
+      ]
+    },
+    "DevBoxActionsDelayMultipleResult": {
+      "type": "object",
+      "description": "The actions list result",
+      "properties": {
+        "value": {
+          "description": "Current page of results",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DevBoxActionDelayResult"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results."
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "DevBoxActionDelayResult": {
+      "type": "object",
+      "description": "The action delay result",
+      "properties": {
+        "id": {
```

Comment:
May have missed this, but why does the delay result have an `id`? If this is the ID of the action post-delay, would `targetId` be more appropriate?